examples/dev/Dashboard.html

```diff
@@ -16,24 +16,13 @@
         margin: auto;
       }
 
-      button,
-      input {
-        color: green;
-        font-size: 30px;
-        text-align: right;
-        border: 2px solid purple;
-      }
-
```

Comment:
These weird styles are here for a reson: try to emulate someting going on on a page Uppy could be embedded on. To solve the issue we need to add `uppy-u-reset` utility classname to all buttons — in this case to `ErrorButton`.

Reply:
`className="uppy-u-reset uppy-Dashboard-Item-errorDetails"`